source/server/lua/loader/LuaDimensionLoader.cpp

```diff
@@ -38,8 +38,34 @@ void LuaDimensionLoader::loadDimension(const sol::table &table) const {
 	if (biomesObject.valid() && biomesObject.get_type() == sol::type::table) {
 		Dimension &dimension = Registry::getInstance().registerDimension(id, name);
 		dimension.setSky(table["sky"].get<std::string>());
-		dimension.setGravity(table["gravity"].get_or(1.f));
 
+		sol::object dimensionPhysicsObject = table["dimension_physics"];
```

Comment:
You used `player_physics` everywhere but here.

Please **compile and test your code** before commiting.

Reply:
I thought I did compile and test it. My bad. I gotta be more careful.

Reply:
Ah, now I know why I missed the fact that it wasn't working. I only tested the default values and forgot to test if lua values worked.